docs/reference/indices/aliases.asciidoc

```diff
@@ -252,7 +252,7 @@ indices that match this pattern are added/removed.
 
 It is an error to index to an alias which points to more than one index.
 
-It is also possible to swap an index with an alias in one operation:
+It is also possible to swap an index with an alias in one, atomic operation:
```

Comment:
@tugberkugurlu, you are correct that this operation is atomic and it would be good to make this explicit in the documentation.